js-pkg/react/src/main.tsx

```diff
@@ -250,38 +241,15 @@ export function YDocProvider(props: YDocProviderProps) {
   const [ctx, setCtx] = useState<YjsContextType | null>(null)
 
   useEffect(() => {
-    let canceled = false
-    let provider: YSweetProvider | null = null
     const doc = new Y.Doc()
+    const provider = createYjsProvider(doc, docId, authEndpoint, {
+      initialClientToken,
+      offlineSupport: props.offlineSupport,
+    })
```

Comment:
Do we need to pass in `showDebuggerLink` here now?

Reply:
whoops good catch